exchanges/credentials.go

```diff
@@ -117,7 +117,8 @@ func (b *Base) GetCredentials(ctx context.Context) (*account.Credentials, error)
 		if !ok {
 			// NOTE: Return empty credentials on error to limit panic on
 			// websocket handling.
-			return &account.Credentials{}, errContextCredentialsFailure
+			return nil, fmt.Errorf("context credentials store type assertion failed for %T: %w",
```

Comment:
Returning nil causes panics in the CI builds, can you please rectify them. Can also change the error wrapping to `fmt.Errorf("%w: %w", common.GetTypeAssertError(params...),errContextCredentialsFailure)`

Reply:
Oh, crap I wasn't supposed to return nil (since the comment mentioned that)

*** I should remove tabnine from my IDE

Reply:
Can leave it as nil but you will need to do a bit more work. What you have done is more correct on what should happen. I will leave it up to you on what you want to do.